app/app.go

```diff
@@ -1212,8 +1344,8 @@ func (a *App) ModuleAccountsPermissions() map[string][]string {
 
 func (a *App) registerUpgradeHandlers() {
 	a.UpgradeKeeper.SetUpgradeHandler(
-		mv10.UpgradeName,
-		mv10.CreateUpgradeHandlerV10(a.mm, a.configurator, a.LiquidityKeeper, a.AssetKeeper, a.BankKeeper, a.AccountKeeper, a.Rewardskeeper, a.ICAHostKeeper),
+		tv11.UpgradeName,
+		tv11.CreateUpgradeHandlerV11(a.mm, a.configurator),
 	)
```

Comment:
Do we need to set the Upgrade handler for mv11 in the current release or will we raise another PR for the same?